bin/migrate_airflow_db.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+
+echo "
+Description:
+  This script is PoC of how can you migrate
+  give airflow deployment to another DB.
+
+  This script assumes:
+
+  Assumption 1: user is responsible of making any networking connectivity between
+  PG server 2 and airflow deployment.
+
+  Assumption 2: you have access to both DB instanceses so you can run pg_dump pg_restore
+
+  Assumption 3:  Make sure you pg_dump/pg_restore versions matched
+"
+
+
+export PATH="/usr/local/opt/postgresql@11/bin:$PATH"
+ASTRONOMER_AIRFLOW_CHART="astronomer-ee/airflow"
+RELEASE_NAME="${1:-dynamical-nucleus-9402}"
```

Comment:
We should not have a default value for RELEASE_NAME
```suggestion
RELEASE_NAME="$1"
```